hack/scripts/olm-nightly-install.sh

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env bash
+
+set -e
+
+if [ ! ${IIB_ID} ] ;
+then
+    echo -e "\nPlease set the environment variables IIB_ID and NEW_VER\n"   
+    exit
+fi
+
+INDEX="brew.registry.redhat.io/rh-osbs/iib:$IIB_ID"
```

Comment:
Can you take in the registry/repository name as an environment variable with default values. `brew.registry.redhat.io/rh-osbs` - REGISTRY_NAME
`iib` - REPO_NAME